apps/web/components/Embed.tsx

```diff
@@ -629,9 +632,9 @@ const ChooseEmbedTypesDialogContent = () => {
   const { t } = useLocale();
   const router = useRouter();
   return (
-    <DialogContent type="creation" size="lg">
-      <div className="mb-4">
-        <h3 className="text-lg font-bold leading-6 text-gray-900" id="modal-title">
+    <DialogContent className="rounded-lg p-10" type="creation" size="lg">
```

Comment:
Hey @sean-brydon, we moved the style changes here. Padding is still `p-10` as per the design
![Screenshot 2023-03-20 at 12 08 09 PM](https://user-images.githubusercontent.com/121884634/226262588-494c9531-9766-4a9e-acdf-3dfc5a900724.png)